test/integration-test/src/tests/load.rs

```diff
@@ -13,6 +15,27 @@ use aya::{
 const MAX_RETRIES: usize = 100;
 const RETRY_DURATION: time::Duration = time::Duration::from_millis(10);
 
+#[test]
+fn tc_name_limit() {
+    let _ = qdisc_add_clsact("lo");
```

Comment:
why is it ok to drop this value?

Reply:
We need the `clsact` qdisc attached to the interface before we can proceed, Actually a better approach would be a function that checks that so we can call `qdisc_add_clsact("lo")` only when that interface doesn't have that qdisc.
I can add that and update the code...

Reply:
@tamird I have added a function to check if `clsact` qdisc exists before trying to actually add it.
I'd really appreciate it if you review the code.

Reply:
> Is there a different type of probe we can use that doesn't have this requirement?

By requirement, Do you mean the need to call `qdisc_add_clsact("lo")`?
If yes, I don't think so.
As far as I know we can either use `ingress` or `clsact` if attaching to ingress path and to attach an ebpf program to egress path, the only option is to use `clsact`:
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=1f211a1b929c804100e138c5d3d656992cfd5622